src/index.html

```diff
@@ -9,6 +12,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <img class="img" src="images/logo.png"
+      alt="logo">
```

Comment:
the alternative text in this case is not very informative
it will be better _Logotype of MOYO_

Reply:
also, keep your attributes correctly formatted, check examples [here](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-content)